examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/EasyJsonObjectRequest.java

```diff
@@ -0,0 +1,44 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.util.Log;
+
+import com.android.volley.NetworkResponse;
+import com.android.volley.ParseError;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.HttpHeaderParser;
+import com.android.volley.toolbox.JsonObjectRequest;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.UnsupportedEncodingException;
+
+public class EasyJsonObjectRequest extends JsonObjectRequest {
+
+    private static final String TAG = "EasyJsonObjectRequest";
```

Comment:
This is a matter of preference, but if you used `EasyJsonObjectRequest.class.getName()` instead of the hardcoded String, refactoring tools (IntelliJ) would catch it if you ever renamed this class.